tasks/compose_Amazon.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Setup for installing Docker compose v2 on Amazon from GitHub
```

Comment:
I don't see anything Amazon-specific in this file (although it is only applied to Amazon systems).
```suggestion
# Setup for installing Docker compose v2 from GitHub
```